pkg/profiles/profiles.go

```diff
@@ -372,8 +372,12 @@ func Validate(data []byte) error {
 
 	rb := serviceProfile.Spec.RetryBudget
 	if rb != nil {
-		if rb.MinRetriesPerSecond == 0 || rb.RetryRatio == 0 || rb.TTL == "" {
-			return fmt.Errorf("ServiceProfile \"%s\" RetryBudget missing fields (requires minRetriesPerSecond, retryRatio, ttl)", serviceProfile.Name)
+		if rb.RetryRatio < 0 {
+			return fmt.Errorf("ServiceProfile \"%s\" RetryBudget RetryRatio must be greater than zero: %f", serviceProfile.Name, rb.RetryRatio)
+		}
+
```

Comment:
We should also validate that MinRetriesPerSecond is non-negative also

Reply:
`MinRetriesPerSecond` is a `uint32` so will be validating via `yaml.UnmarshalStrict`:
```
failed to validate ServiceProfile: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal number -5 into Go struct field RetryBudget.minRetriesPerSecond of type uint32
```

Reply:
fantastic!